Added the html file for the mediterranean project

mikaelk opened the pull request:
A pull request to add the mediterranean sea project to the TOPIOS.org website. The only thing I am not sure about how to link to the mediterranean.html page from index.html: it is now a stand-alone page. Should we e.g. add a link to the navigation bar?

mikaelk committed:
Added the html file for the mediterranean project

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+    </p><p>
+    A model of floating particles is made to represent floating plastics as accurately as possible, by comparing it with these plastic concentration measurements. Results from the model show that the total plastic input into the Mediterranean sea likely ranges between 2,200-4,000 tonnes for the year 2015, that only a small part of the plastics remain afloat (about 240-390 tonnes), that about half ...
+    </p><p style="color:#9E9E9E">
+    	<small>author: Mikael Kaandorp, date: May 2020</small>
```
Make this a link to either your UU staff page, your ORCiD or your github profile, whichever you prefer?

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+  <h1 class="display-3" style="text-align: center;">Closing the Mediterranean Floating Plastic Mass Budget</h1>
+	
+	<p>
+	It is currently not well known how much plastics are entering our oceans, and what happens with these plastics in the environment. Most plastics are lighter than water, and are therefore expected to float. However, over time a layer of algae increases the weight of these plastic particles, possibly leading to these particles sinking down. Another significant part of plastics might eventually end ...
```
I'm not sure the plural 'plastics' works well here. Normally, that is used for chemical types of plastic, just like 'fishes' is multiple species of the already plural 'fish'. 
So using 'plastics' here might suggest that you analyse different types of plastic, which you don't. I would use 'plastic' throughout, hence

mikaelk replied:
done; changed to 'plastic' or 'plastic particles'

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+	<p>
+	It is currently not well known how much plastics are entering our oceans, and what happens with these plastics in the environment. Most plastics are lighter than water, and are therefore expected to float. However, over time a layer of algae increases the weight of these plastic particles, possibly leading to these particles sinking down. Another significant part of plastics might eventually end ...
+	</p><p>
+	Here, we developed a method to estimate the total input of plastics in the Mediterranean, and where these plastics are expected to end up. The Mediterranean is a nice test case: it is confined, a lot of people are living in this area, which likely means a lot of plastic pollution as well, and there are many plastic concentration measurements available from the surface water.
```
is 'nice' the best word? Would 'good' not be less emotive?

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+	</p><p>
+	Here, we developed a method to estimate the total input of plastics in the Mediterranean, and where these plastics are expected to end up. The Mediterranean is a nice test case: it is confined, a lot of people are living in this area, which likely means a lot of plastic pollution as well, and there are many plastic concentration measurements available from the surface water.
+    </p><p>
+    A model of floating particles is made to represent floating plastics as accurately as possible, by comparing it with these plastic concentration measurements. Results from the model show that the total plastic input into the Mediterranean sea likely ranges between 2,200-4,000 tonnes for the year 2015, that only a small part of the plastics remain afloat (about 240-390 tonnes), that about half ...
```
```suggestion
    A model of floating particles is made to represent floating plastics as accurately as possible, by comparing it with these plastic concentration measurements. Results from the model show that the total plastic input into the Mediterranean Sea likely ranges between 2,200-4,000 tonnes for the year 2015, that only a small part of the plastics remain afloat (about 240-390 tonnes), that about half of the plastics start sinking down, and that about half of the plastics are expected to end up on the coastlines.
```
Use capital S for Mediterranean Sea

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+      <p class="card-text">
+		Floating plastics are simulated using a Lagrangian model: this means virtual particles, representing plastics, are released in a virtual ocean. 
+      </p><p>
+		The particles are moved around using reanalysis data for the currents [1], and influence of the waves (Stokes drift) [2]. The simulation is run between 2006-2016. 
```
Make all of these references links?

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+  
+  <hr>
+
+  <h2 id="measurements">Does the model represent reality?</h2>
```
This is not really the question here, right? instead of validating the model, we 'tune' the model with real observation

mikaelk replied:
changed the section title to 'Using measurements to find our model parameters'

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+
+  <h2 id="conclusion">Conclusion and outlook</h2>
+  <div class="row">
+    <div class="col-lg-9" style="margin-bottom: 20px">
```
Why not span this last section over the full page width, like the first section?

erikvansebille commented on docs/mediterranean.html:
```diff
@@ -0,0 +1,241 @@
+
+  <hr>
+
+  <h2 id="conclusion" style="color:#9E9E9E">References</h2>
```
Second use of `id="conclusion"`?

erikvansebille reviewed (commented):
Nice work @mikaelk! See some comments on the html inline. 

Some other points:
- [x] add some text that relates this work to the overall TOPIOS project?
- [x] add links to the subsections on this page in the top-right header (like on topios.org)?
- [x] the youtube videos don't scale very well if the page width is reduced (like e.g. on a mobile phone)
- [x] add a link to this page from index.html. Not entirely sure how. Options are:
    1. Add a 'paper card' to http://topios.org/#peerreviewedarticles
    2. Add an image plus link to the Carousel on the top
    3. Create a new section 'Project highlights' after Team and before Articles
    4. ...

mikaelk committed:
Update docs/mediterranean.html

Co-authored-by: Erik van Sebille <erikvansebille@gmail.com>

mikaelk committed:
Modified index.html and mediterranean.html with suggested changes from Erik

- youtube embeds scale with width
- relation to TOPIOS project
- link from carousel index.html to mediterranean.html

mikaelk committed:
added mediterranean image for carousel

mikaelk commented:
Hi Erik, thanks for the comments! I adjusted where necessary: a introduction was added relating the project to TOPIOS, references were changed to links, youtube embedded videos now scale with width, I made a link from index.html to mediterranean.html using the Carousel.
